src/workflow.ts

```diff
@@ -64,7 +65,7 @@ export interface GetWorkflowsInput {
   authenticatedUser?: string; // The user who ran the workflow.
   startTime?: string; // Timestamp in RFC 3339 format
   endTime?: string; // Timestamp in RFC 3339 format
-  status?: "PENDING" | "SUCCESS" | "ERROR"; // The status of the workflow.
+  status?: "PENDING" | "SUCCESS" | "ERROR" | "DEADLETTER"; // The status of the workflow.
```

Comment:
Is there a better name than "deadletter"? Like "NORETRY" or something?

Reply:
Sure, I can change that